test/gurka/gurka.cc

```diff
@@ -464,7 +464,7 @@ map buildtiles(const nodelayout& layout,
   detail::build_pbf(result.nodes, ways, nodes, relations, pbf_filename);
   std::cerr << "[          ] building tiles in " << result.config.get<std::string>("mjolnir.tile_dir")
             << std::endl;
-  midgard::logging::Configure({{"type", ""}});
+  midgard::logging::Configure({{"type", "std_out"}});
```

Comment:
you might have accidently left this on for testing? honestly i kind of like this change now that the logging is vastly reduced for route requests, however the logging for tile creation is still very verbose (as it should be)